framework/DeclManager.cpp

```diff
@@ -1028,7 +1028,7 @@ void idDeclManagerLocal::LoadManual( )
 				pageB = 0;
 			}
 
-			f->Printf( va("skin manualpage_%d_%d\n{\nmodels/weapon_map/page_%d  %s\nmodels/weapon_map/page_%d  %s\n}\n\n",
+			f->Printf( "%s", va("skin manualpage_%d_%d\n{\nmodels/weapon_map/page_%d  %s\nmodels/weapon_map/page_%d  %s\n}\n\n",
```

Comment:
Something might be wonky about this line.  And if so, that probably means that https://github.com/blendogames/quadrilateralcowboy/pull/4/files#diff-06d242b230b879c302017db72bb1abda459ca45630fb0a4658be6362bc3aa651R3260 is wonky too.

Reply:
Yeah, it's a case of using va() in Printf(), so the right thing to do would be to remove the added `"%s",` and `va(` and one `)` at the end to use Printf directly